integration_tests/dbt_project.yml

```diff
@@ -20,15 +20,18 @@ vars:
     zendesk_domain_name_identifier:            "domain_name_data"
     zendesk_time_zone_identifier:              "time_zone_data"
     zendesk_daylight_time_identifier:          "daylight_time_data"
-    group:                                     "{{ ref('group_data') }}" 
-    brand:                                     "{{ fivetran_utils.seed_data_helper('brand_data',['postgres']) }}" 
-    user_tag:                                  "{{ fivetran_utils.snowflake_seed_data('user_tag_data') }}" 
-    organization_tag:                          "{{ fivetran_utils.snowflake_seed_data('organization_tag_data') }}" 
-    user:                                      "{{ fivetran_utils.snowflake_seed_data('user_data') }}" 
+    zendesk_brand_identifier:                  "brand_data"
+    zendesk_group_identifier:                  "group_data" 
+    zendesk_organization_tag_identifier:       "organization_tag_data"
+    zendesk_user_identifier:                   "user_data"
+    zendesk_user_tag_identifier:               "user_tag_data"
 
 seeds:
-  +quote_columns: "{{ true if target.type == 'redshift' else false }}"
+
   zendesk_integration_tests:
+    +quote_columns: "{{ true if target.type == 'redshift' else false }}"
```

Comment:
why is there a condition on redshift?

Reply:
It appears to be an artifact of our CircleCI integration. I'm not sure if we still need it but at the moment it isn't really blocking anything, so we'll discuss it internally. Thanks for spotting!